class/spec/student_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'spec_helper'
+
+describe Student do
+  before :each do
+    @student = Student.new('Unknown')
+  end
+
+  describe '#new' do
+    context 'when creating a student instance' do
+      it 'returns a student object' do
+        expect(@student).to be_an_instance_of Student
+      end
+
+      it 'returns the students name' do
+        expect(@student.name).to eql 'Unknown'
+      end
+    end
+  end
+end
```

Comment:
- [ ] Please remember to test all methods in a class, currently, you did not add a unit test for the method `play_hooky` kindly include that

Reply:
Are these the only changes

Reply:
yes please

Reply:
This test has been added well